sa/proto/sa.proto

```diff
@@ -71,6 +74,10 @@ service StorageAuthority {
   rpc SerialsForIncident (SerialsForIncidentRequest) returns (stream IncidentSerial) {}
   rpc CheckIdentifiersPaused (PauseRequest) returns (Identifiers) {}
   rpc GetPausedIdentifiers (RegistrationID) returns (Identifiers) {}
+  rpc GetRateLimitOverride(GetRateLimitOverrideRequest) returns (RateLimitOverrideResponse) {}
+  rpc GetEnabledRateLimitOverrides(google.protobuf.Empty) returns (stream RateLimitOverride) {}
+  rpc SearchRateLimitOverrides(SearchRateLimitOverridesRequest) returns (stream RateLimitOverrideResponse) {}
```

Comment:
I think it's not necessary to include the Search method in this first PR, for a few reasons:
- It won't be used by the admin tool to load existing overrides
- It won't be used by the folks who talk to our subscribers the most, since they don't have admin/gRPC access
- If the idea is that it will be useful for investigating recent auto-approved override changes, it won't be necessary until the auto-approval infrastructure exists, which is several PRs and deploys from now

Also, without having the full auto-approval pipeline and admin-based investigation tooling in place yet, it's really hard to tell if the search parameters exposed by this are the ones we'll really end up wanting. So I'd save this for later.